### PR TITLE
Domains: Update example state to be more helpful.

### PR DESCRIPTION
**Pull request by shaunandrews**
This PR aims to simplify the default "blank state" for domain search. Here's the current design:

screen shot 2016-08-02 at 4 16 39 pm

Here's the update design which aims to answer the simple question "What is a domain" and encourage people to start searching, rather than processing example options:

screen shot 2016-08-03 at 3 15 55 pm

### CODE REVIEW

File: client/components/domains/example-domain-suggestions/index.jsx [hunk 1]
```diff
@@ -2,139 +2,33 @@
...
-						{ this.translate( 'What are my options?' ) }
-					</h2>
-					<ul className="example-domain-suggestions__list">
-						{ this.getExampleSuggestions() }
-					</ul>
-					<p className="example-domain-suggestions__mapping-information">
-						{ mappingInformation }
-					</p>
+			<div className="example-domain-suggestions">
+				<p className="example-domain-suggestions__explanation">A domain name is what people type into their browser to visit your site.</p>
```
Comment (klimeryk) [inline client/components/domains/example-domain-suggestions/index.jsx]: This string (as well as the one below) should be wrapped in `this.translate` to make them, well, translatable :)

File: client/components/domains/example-domain-suggestions/style.scss [hunk 2]
```diff
@@ -1,88 +1,26 @@
...
-.example-domain-suggestions__mapping-information {
-	color: darken( $gray, 20% );
-	margin: 16px 24px;
 
 	a {
 		text-decoration: underline;
 	}
 }
+
+.example-domain-suggestions__mapping-information {}
```
Comment (klimeryk) [inline client/components/domains/example-domain-suggestions/style.scss]: Is there a need for this style anymore?

**Comment by folletto** [comment]
Design wise: 👍  :shipit:

**Comment by klimeryk** [comment]
Thanks for addressing my comments! :bow:
Code looks and works good 👍 
But I've noticed there's some extra, unnecessary space between `http://` and `example.com` (there's none in your screenshots):

Chromium 51.0.2683.0:
screen shot 2016-08-04 at 19 00 31

Safari 9.1.2 (11601.7.7):
screen shot 2016-08-04 at 19 01 07

**Comment by klimeryk** [comment]
Thanks - interesting workaround :) The text looks fine now.

But now I've noticed that for smaller screens, the browser placeholder looses the, umm, fade out?

screen shot 2016-08-04 at 20 32 29
vs
screen shot 2016-08-04 at 20 32 38

Sorry for nitpicking like this, not sure if this is helpful to you or just annoying 😆 Just trying to make sure the designs are in line with the implementation :)

**Comment by shaunandrews** [comment]
Ah, good catch @klimeryk, I fixed the missing fade overlay. Should be good to go.

**Comment by klimeryk** [comment]
Sweet, thanks! Everything looks A-OK now 👍

**Pull request merged**